README.md

```diff
@@ -0,0 +1,86 @@
+# Static Analysis Gradle Plugin
+
+1. [Overview](#overview)
+1. [Usage](#usage)
+1. [Configuration](#configuration)
+1. [License](#license)
+
+### Overview
+The Static Analysis Gradle Plugin adds gradle tasks for running PMD and FindBugs, and includes a default configuration for these along with Android Lint that is common to most Intrepid Android projects.
+
+### Usage
+
+Add the plugin to your project:
+```
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath "gradle.plugin.io.intrepid:static-analysis:1.0"
+    }
+}
+
+apply plugin: "io.intrepid.static-analysis"
+```
+
+Or alternatively, if you are using Gradle 2.1+, add the following to the top of your app `build.gradle`:
+```
+plugins {
+    id "io.intrepid.static-analysis" version "1.0"
+}
+```
+
+To run pmd, use the command `./gradlew pmd`.
+
+To run FindBugs, use the command `./gradlew findBugs${buildVariant}`. By default, this produces a html result. You can change it to produce a xml report instead by adding `findBugsXml` flag (FindBugs can only produce one type of report at a time).
+ex: `./gradlew findBugsDebug -PfindBugsXml`
+
+Android Lint can be run using the standard `./gradlew findBugs${buildVariant}`
```

Comment:
If this changes the default lint behavior, we should be crystal clear about that and exactly how it does so.